python/llm/example/CPU/HF-Transformers-AutoModels/Model/glm-4v-9b/generate.py

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+
+
+import os
+import time
+import torch
+import argparse
+import requests
+
+from PIL import Image
+from ipex_llm.transformers import AutoModelForCausalLM
+from transformers import AutoProcessor, AutoTokenizer
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Predict Tokens using `generate()` API for THUDM/glm-4v-9b model')
+    parser.add_argument('--repo-id-or-model-path', type=str, default="THUDM/glm-4v-9b",
+                        help='The huggingface repo id for the THUDM/glm-4v-9b model to be downloaded'
+                             ', or the path to the huggingface checkpoint folder')
+    parser.add_argument('--image-url-or-path', type=str,
+                        default="http://farm6.staticflickr.com/5268/5602445367_3504763978_z.jpg",
+                        help='The URL or path to the image to infer')
+    parser.add_argument('--prompt', type=str, default="What is in the image?",
+                        help='Prompt to infer')
+    parser.add_argument('--n-predict', type=int, default=32,
+                        help='Max tokens to predict')
+
+    args = parser.parse_args()
+    model_path = args.repo_id_or_model_path
+    image_path = args.image_url_or_path
+    device = "cpu"
+    
+    model = AutoModelForCausalLM.from_pretrained(model_path,
+                                                 load_in_4bit=True,
+                                                 low_cpu_mem_usage=True,
+                                                 trust_remote_code=True).to(device)
```

Comment:
Maybe we could use the keyword arguments as we did here: https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/example/CPU/HF-Transformers-AutoModels/Model/llama3/generate.py#L58-L62

`low_cpu_mem_usage=True` seems to be a default argument for us.  You could have a try that whether add it or not will cause any change in memory.

Reply:
Same for GPU example :)